maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/ValidationService.java

```diff
@@ -39,34 +39,11 @@ public class ValidationService {
     public static final String USER_DOES_NOT_HAVE_A_VALID_NEW_WORK_REASON_CODE =
             "User does not have a valid New Work Reason Code";
 
-    public void validate(WorkflowRequest request, RepOrderDTO repOrderDTO) {
-        validateApplicationTimestamp(request, repOrderDTO);
-        validateApplicationStatus(request, repOrderDTO);
-    }
-
-    private void validateApplicationTimestamp(WorkflowRequest request, RepOrderDTO repOrderDTO) {
-        ZonedDateTime applicationTimestamp = request.getApplicationDTO().getTimestamp();
-
-        if (applicationTimestamp == null) {
-            return;
-        }
-
-        LocalDateTime repOrderCreatedDate = DateUtil.convertDateToDateTime(repOrderDTO.getDateCreated());
-        LocalDateTime repOrderUpdatedDate = repOrderDTO.getDateModified();
-        LocalDateTime repOrderTimestamp = (null != repOrderUpdatedDate) ? repOrderUpdatedDate : repOrderCreatedDate;
+    public void checkUpliftFieldPermissions(final UserSummaryDTO userSummaryDTO) {
+        boolean userCanEditUpliftApplied = this.isUserAuthorisedToEditField(userSummaryDTO, RestrictedField.UPLIFT_APPLIED);
```

Comment:
Is the intention to have a separate method that compares the `ApplicationDTO` with what's stored in the `RepOrder` table e.g. `checkFieldPermissions`? If so I think we should probably add this as part of the current PR, which we can continue to flesh out as we come across more restricted fields during the migration.

Reply:
Do you mean, is the intention to have a separate method that compares the application passed in from the _WorkflowRequest_ against the existing application? Apologies for not quite understanding the question here

Reply:
Yeah, that's right :)

Reply:
Gotcha. Given this PR has been in the approved state for a week and also given that the above wasn't part of the description of the Jira task, I'm going to merge this in as-is. Happy to add a follow-up task to Jira to capture what you've said here and work on that too.